Code/Engine/Audio/Audio.hpp

```diff
@@ -53,6 +55,8 @@ class AudioSystem
     void StopSound(SoundID soundID);
     void StopChannel(AudioChannelHandle channel);
 
+    void CreateDSPByType(FMOD_DSP_TYPE type, FMOD::DSP** dsp);
```

Comment:
`FMOD::DSP` needs to be replaced with `DSPHandle` in this header, since we're not exporting that type to anyone else.